tests/operators/test_agnostic_save_file.py

```diff
@@ -290,6 +290,40 @@ def test_save_bigquery_table_to_local_file_exists_overwrite_false(self):
         # Delete output file after run
         os.remove(OUTPUT_FILE_PATH)
 
+    def test_save_sqlite_table_to_local_file_exists_overwrite_false(self):
```

Comment:
An alternative to adding 22 new lines of code could be to add 'sqlite'  as a database in the `test_save_file`? I noticed it is also missing BQ, it would be great if it could be covering all the databases supported! I already had adjusted the load_file one to include BQ, so it should be simple :crossed_fingers: 
https://github.com/astro-projects/astro/pull/119/files#diff-5692c92b876e45ee085b44ef344a19a0c40b9d29af9c03de199e98dff0fe8c0eR380

Reply:
Sure, I'll update the PR